test/unit/controllers/ChoiceControllerSpec.scala

```diff
@@ -87,23 +87,21 @@ class ChoiceControllerSpec extends ControllerSpec with OptionValues with BeforeA
       }
     }
 
-    "return 400 (BAD_REQUEST)" when {
+    "throw an IllegalArgumentException" when {
 
       "form is incorrect" in {
 
-        val incorrectForm = Json.toJson(Choice("incorrect"))
-
-        val result = controller.submitChoice()(postRequest(incorrectForm))
-
-        status(result) mustBe BAD_REQUEST
+        intercept[IllegalArgumentException] {
+          await(controller.submitChoice()(postRequest(Json.toJson(Choice("incorrect")))))
```

Comment:
same - not exception but badRequest od as @PauloPortugal point [422 Unprocessable Entity](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422)

Reply:
Fixed